lib/Installation/ProductSelection/ProductSelectionPage.pm

```diff
@@ -34,7 +34,7 @@ sub init {
 
 sub is_shown {
     my ($self) = @_;
-    return $self->{rdb_SLES}->exist();
+    return $self->{rdb_SLES}->exist({timeout => 300, interval => 10});
```

Comment:
we don't need to have this for every single product selection, I know you use it to probe that his works, but we are probably increasing the timeout where is not needed to 300, hiding a potential bug.

Reply:
how about just 300 for Aarch64 for the others use the default(120)?

Reply:
we might have done something like this in a rush in the past, but the normal way is to put the timeout from outside, in the test layer, and pass these two parameters through the layer to the is_shown to here when needed, so we can multiply normal timeout by some number. We don't do those hacks of hide conditions in this framework " in general " like it is done over the place for testapi.

Reply:
I'd think of a way to pass the timeout from the outer to the inner part as you described.

Reply:
ok, you want to do the extra mile :)
check for example: `installation/performing_installation/perform_installation.pm`

Reply:
the is_shown was controlled by NavigationBase, and called before the  install_product. so we can't pass timeout from the outer install_SLE.pm.
so I just add TIMOUT_SCALE in  ProductSelectionPage.